src/managers/init_manager.ts

```diff
@@ -154,25 +154,15 @@ export default class InitManager {
     return this.addMissingUserFiles(dataPath);
   }
 
-  public static checkForKey(object: any, keyPath: string[], key: string): boolean {
```

Comment:
What happened to this function? Is it not used anywhere?

Reply:
Doesn't look like it was, also doesn't look like it was working properly